### PR TITLE
add kubernetes version cri version output in sealos verison cmd

### PR DESCRIPTION
**Pull request by cdjianghan**

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -35,3 +43,35 @@ func Get() Info {
 		Platform:   fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+func GetKubernetesVersion(cluster *v2.Cluster) *KubernetesVersion {
```
Comment (zzjin) [inline pkg/version/version.go:47]:
```suggestion
func GetKubernetesVersion(cluster *v2.Cluster) (*KubernetesVersion, error) {
```
return error if cmd exec fail or version info is mismatch when set debug=true,and print it out.

Do not omit any errors.

Comment (fengxsong) [reply pkg/version/version.go:47]: > return error if cmd exec fail or version info is mismatch when set debug=true,and print it out.
> 
> Do not omit any errors.

but what if `kubectl` or `crictl` binary are missing?

Comment (zzjin) [reply pkg/version/version.go:47]: After discussion, will show not found and suggest user to start kubernetes cluster using `sealos run`.
image

File: pkg/version/types.go [hunk 2]
```diff
@@ -16,21 +16,61 @@ limitations under the License.
...
-	BuildDate  string `json:"buildDate"`
-	GoVersion  string `json:"goVersion"`
-	Compiler   string `json:"compiler"`
-	Platform   string `json:"platform"`
+	GitVersion string `json:"gitVersion" yaml:"gitVersion"`
+	GitCommit  string `json:"gitCommit,omitempty" yaml:"gitCommit,omitempty"`
+	BuildDate  string `json:"buildDate" yaml:"buildDate"`
+	GoVersion  string `json:"goVersion" yaml:"goVersion"`
+	Compiler   string `json:"compiler" yaml:"compiler"`
+	Platform   string `json:"platform" yaml:"platform"`
```
Comment (lingdie) [inline pkg/version/types.go:32]: why add yaml?

Comment (cdjianghan) [reply pkg/version/types.go:32]: output can use “-o yaml | json” to define output format

Comment (lingdie) [reply pkg/version/types.go:32]: But in k8s crd, we can '-o yaml' to format output too, and we don't re-define `yaml` filed, what's the difference?

Comment (cdjianghan) [reply pkg/version/types.go:32]: no difference ,just for format convert convience.
another point is crictl dont have -o options

**Review by zzjin — APPROVED** [review]

**Pull request merged**